lib/players/computer.rb

```diff
@@ -0,0 +1,10 @@
+require 'pry'
+
+class Computer < Player
+  def move(board)
+    sleep 0.3
```

Comment:
Player: "Computer, are you asleep?"
Computer: "No no, just thinking with my eyes closed."